### ISSUE TITLE
Performance of fromBase64

### ISSUE THREAD

**Issue by ceedubs** [scodec/scodec-bits#32]
This issue is similar to #29 but going in the opposite direction: base-64 encoded String to Bit/ByteVector.

My benchmarks are showing that it is much faster to use `java.util.Base64` to decode into an `Array[Byte]` and then create a `BitVector` from that than it is to use `BitVector.fromBase64` directly.

```
[info] Benchmark                             Mode  Cnt    Score   Error  Units
[info] FooBenchmark.bitVectorFromBase64      avgt  200  681.362 ± 3.274  us/op
[info] FooBenchmark.bitVectorFromBase64Java  avgt  200    2.870 ± 0.019  us/op
```

**Comment by ceedubs** [comment]
Sorry I forgot to actually link the benchmark: https://github.com/ceedubs/scodec-benchmarks/blob/7d69d55fa3b73e9db36fd7469957b83d1917757b/src/main/scala/FooBenchmark.scala#L49-L53

**Comment by mpilquist** [comment]
Fixed by #40. If you get a chance to run benchmarks against 1.0.10-SNAPSHOT, let me know how it looks.

**Comment by ceedubs** [comment]
@mpilquist thanks!

In the benchmarks that I linked to, here are the results I get now:

```
[info] Benchmark                             Mode  Cnt  Score   Error  Units
[info] FooBenchmark.bitVectorFromBase64      avgt  200  3.168 ± 0.007  us/op
[info] FooBenchmark.bitVectorFromBase64Java  avgt  200  2.846 ± 0.005  us/op
```

However, this isn't quite a fair comparison, because the JRE version throws exceptions that would need to be caught and turned into an Option to compare to the scodec version. When I do that, the JRE version still wins by a hair, but they are in the same ballpark. This is definitely a huge win from the previous performance!
